src/python/turicreate/data_structures/sframe.py

```diff
@@ -2188,7 +2188,13 @@ def _imagecols_to_stringcols(self, num_rows=10):
                 if t in image_column_names:
                     printed_sf[t] = self[t].astype(str)
         return printed_sf.head(num_rows)
-
+    
+    def drop_duplicates(self, subset): 
+        result =  all(elem in self.column_names()  for elem in subset)
```

Comment:
Please add a docstring comment. This should explain what `drop_duplicates` does and also the meaning of `subset`.

Reply:
Once we get a doc string in here (ideally with a few examples), this is good to merge.